pkg/api/controller.go

```diff
@@ -3493,6 +3493,39 @@ func (c *Controller) GetTag(w http.ResponseWriter, r *http.Request, repository s
 	writeResponse(w, r, http.StatusOK, response)
 }
```

Comment:
Regarding [logout_handler](https://github.com/treeverse/lakeFS/blob/master/pkg/api/logout_handler.go#L11) handler, if we handle some logic in lakeFS and only then redirect to external provider (instead of directly redirecting from UI to logout provider skipping lakeFS) it's also need to be modified.

Reply:
We're already aligned, maybe the OIDC should be removed in the future, but nothing else.